subscriptions/subscriptions-impl/src/main/java/com/duckduckgo/subscriptions/impl/settings/views/PirSettingViewModel.kt

```diff
@@ -63,8 +63,6 @@ class PirSettingViewModel(
             subscriptions.getEntitlementStatus(PIR_PRODUCT_NAME).also {
                 if (it.isSuccess) {
                     _viewState.emit(viewState.value.copy(hasSubscription = it.getOrDefault(NotFound) == Found))
-                } else {
```

Comment:
Removing this and leaving it as it was before. In case of failure the state won't change. Before bot being authenticated was considered a failure but not anymore.